### DISCUSSION TITLE
Docker "denied" error when following tutorial.

### DISCUSSION THREAD

**Discussion by metasim** [cerbos/cerbos#1485] [Q&A] [answered]
I'm on the _Running Locally_ step of the tutorial, and when I run

```
docker run --rm --name cerbos -t \
  -v $(pwd)/policies:/policies \
  -p 3592:3592 \
  ghcr.io/cerbos/cerbos:latest server
```

I get 

```
docker: Error response from daemon: Head "https://ghcr.io/v2/cerbos/cerbos/manifests/latest": denied: denied.
```

Is there a public version of this image available, or is there a step missing?

**Comment by charithe** [comment]
Hi. The image is public. It must be a temporary issue with GitHub. We'll investigate.

**Comment by charithe** [comment]
Looks like a temporary blip in the GitHub container registry. It seems to be resolved now. Please try again.

**Comment by metasim** [comment, marked as answer]
Thanks for the responses. The solution had something to do with my previously used ghcr.io credentials. This fixed it:

```shell
docker logout ghcr.io
```

Solution source: https://stackoverflow.com/a/75304123/296509
